ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -683,8 +683,10 @@ def set_codeowners_of(cls, location, span=None):
             handles = cls._instance._codeowners.of(location)
             if handles:
                 span.set_tag(test.CODEOWNERS, json.dumps(handles))
+            else:
+                log.debug("no matching codeowners for %s", location)
         except KeyError:
```

Comment:
Actually, it should just be a bare `except`, really. `of()` [no longer raises an exception itself]( https://github.com/DataDog/dd-trace-py/pull/10094/files#diff-68384ccfde4c133ff7fb33b050476b2fa3efeac1ced057790ef2fe03df140777L195) since #10094 .